db/re/item_db.txt

```diff
@@ -1020,7 +1020,7 @@
 1668,Sword_Stick,Sword Stick,5,10,,500,120:150,,,2,0x800200,63,2,2,4,80,1,10,{ bonus bAspdRate,10; },{},{}
 1669,Thanos_Staff,Thanatos Staff,5,10,,1000,100:200,,1,1,0x00018314,56,2,2,4,120,1,10,{ bonus bInt,6; bonus bVit,6; bonus bLuk,-6; bonus bHealPower,15; bonus bMagicHPGainValue,500; bonus bMagicSPGainValue,50; bonus2 bHPLossRate,100,10000; },{},{ heal -1000,0; }
 1670,RWC_Memory_Staff,RWC Memory Staff,5,20,,500,25:30,,1,1,0x00818315,63,2,2,3,1,1,10,{ .@r = getrefine(); bonus bMatk,30*(.@r/3); if(.@r>=6) bonus2 bMagicAddClass,Class_All,(.@r>=9?10:5); if(.@r>=9) bonus4 bAutoSpell,"HW_MAGICPOWER",1,10,0; },{},{}
-1671,Staff_Of_Evil_Slayer,Devil Won Staff,5,10,,800,30:155,,,1,0x00818315,63,2,2,3,100,1,10,{ bonus2 bAddRace,RC_Undead,10; bonus2 bAddRace,RC_Demon,10; bonus2 bMagicAddRace,RC_Undead,10; bonus2 bMagicAddRace,RC_Demon,10; .@r = getrefine(); if(.@r>=12) bonus bMatkRate,7; else if(.@r>=9) bonus bMatkRate,5; },{},{}
+1671,Staff_Of_Evil_Slayer,Devil Won Staff,5,10,,800,30:155,,,1,0x00818315,63,2,2,3,100,1,10,{ bonus2 bAddRace|MagicAddRace,RC_Undead,10; bonus2 bAddRace|MagicAddRace,RC_Demon,10; .@r = getrefine(); if(.@r>=9) bonus bMatkRate,(.@r>=12)?12:5; },{},{}
```

Comment:
`bonus2` doesn't support bitmask value for `<bonus type>`, only one `bonus type` can be applied at a time

Reply:
@Atemo I didn't know and as it didn't showed error in map-server, so I thought that was working.

Reply:
The item will be updated in https://github.com/rathena/rathena/pull/2555/files#diff-7ae9828022505142aeffdca7376c4412R1023
(but don't ask me when the PR will be merged)
The item script is fine for now. For me this PR can be closed, optimization will come when the PR above will be merged.

Reply:
You're correct, for me its fine too, I didn't see that it had already been fixed after my PR.